.github/workflows/frontend-tests.yml

```diff
@@ -32,14 +32,27 @@ jobs:
     - uses: satackey/action-docker-layer-caching@v0.0.11
       continue-on-error: true
 
+    # We do not use this to install node but only to register problem matchers
+    # so that eslint annotations work.
+    - name: Setup Node.js environment to enable annotations
+      uses: actions/setup-node@v2
+
     - name: Build frontend containers
       run: ./test.sh fe -b
 
     - name: Run frontend tests
       run: ./test.sh fe
 
+    # We run eslint inside a docker container, the frontend sources are mounted
+    # as a volume at /code/static. Therefore, the file name in the eslint output
+    # has file paths beginning with /code/static. As a result, Github Actions is
+    # unable to display those inline with code. To fix, this the output of eslint
+    # is piped to sed which replaces /code/static with listenbrainz/webserver/static .
+    # Finally, piping eats up the exit status so we exit manually with the correct status.
     - name: Run linter
-      run: ./test.sh fe -f
+      run: |
+        ./test.sh fe -f | sed 's/\/code\/static/listenbrainz\/webserver\/static/g'
```

Comment:
Feels very black-magic-ey, and not in a comfortable way, but I don't have any good alternative to offer (short of changing all the paths we use in the docker mounts).

Reply:
Yeah, that was the only alternative I could think of and it seemed not to be much more far reaching.